### ISSUE TITLE
Have render.openstreetmap.org automatically fail over in case servers go down

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#670]
Right now render.openstreetmap.org directs at the two largest European caches, which have the most capacity. Because there are two of them, we should automatically stop using one if it stops responding. One option would be to redirect the render map requests to tile.openstreetmap.org. I don't like this for a few reasons
- they're different services from the user's POV, because one can be used by the public, and the other is on osm.org only
- they're going to different backend servers, since they go to servers with more capacity, and there's no even/odd MT splitting
- the cookie and query parameter handling needs to be completely different, since the totp cookie is used

**Comment by tomhughes** [comment]
We can configure a request count based geodns like we do for nominatim - we just need to decide which machines to include and how to weight them and get some per-country statistics for render requests.

**Comment by pnorman** [comment]
I don't think we really need geodns. We're likely to use machines near each other, and the latency for an export call is a few seconds, so additional network latency isn't a huge issue. Additionally, because each request is unique and uncacheable, there aren't cache locality reasons to split up traffic.
